DOCS/request-new-license.md

```diff
@@ -3,28 +3,29 @@ The SPDX Legal Team appreciates proposals for new free and open source licenses
+    1. New licenses may be approved if 3 SPDX-legal team members (at least 1 lawyer) sign-off that the license is acceptable AND there is no objection raised from the greater SPDX-legal community within the Github issue comments. If there are objections, then the issue will be labeled "discuss on legal call" and will be discussed on an upcoming bi-weekly call.
+    2. New licenses that have already met the criteria to be allowed in [Fedora](https://docs.fedoraproject.org/en-US/legal/license-approval/) or [Debian](https://www.debian.org/social_contract) and code under the license exists in that distro, can be approved by 2 SPDX-legal team members.
   
-2. Issues will be labeled either "new license/exception: Accepted" or "new license/exception: Not Accepted" as appropriate with an explanation and the Issue closed for the latter case.
+2. Issues will be labeled either `new license/exception: Accepted` or `new license/exception: Not Accepted` as appropriate with an explanation and the Issue closed for the latter case.
 
-3. If accepted, the new license or exception will be need to be prepared in the proper XML format and plain text test file via a Pull Request and reviewed as appropriate. The plain text test file should be UTF-8 encoded and match the text and formatting of the original license. The Pull Request is ideally done by the license submitter, with help from experienced members of the SPDX legal team, as needed.
+3. If accepted, the new license or exception files will be need to be prepared via a Pull Request: a plain text test file should be UTF-8 encoded and match the text and formatting of the original license; and an XML file with the appropriate specific tags. 
+    1. It is expected that the Pull Request is prepared by the license submitter, with help from experienced members of the SPDX-legal team, as needed.
+    1. An SPDX-legal team member will review and merge the PR, unless changes are needed.
 
-4. The new license/exception will be officially added for the next release of the SPDX License List.
+4. The new license/exception will be officially added and appear on the SPDX License List website for the next release of the SPDX License List.
 
-Note: Once a license has been accepted to be added to the SPDX License List, it is fine to start using the SPDX identifier, even if the official release has not yet occurred. 
-Note: If a license is *not* accepted for inclusion on the SPDX License List, you can use the 'LicenseRef-[idstring]' tag to identify your license, as per [SPDX specification, section 6](https://spdx.github.io/spdx-spec/6-other-licensing-information-detected/)
+Note: Once a license has been accepted and to be added to the SPDX License List and the associated files merged, it is fine to start using the SPDX identifier, even if the official release has not yet occurred. 
```

Comment:
Possibly add here: "Prior to the official release, the new license and identifier will be visible at [the License List preview site](https://spdx.github.io/license-list-data/)."

Reply:
Not adding for now, we can add later if desired.